cmake/Macros.cmake

```diff
@@ -2,13 +2,14 @@ include(CMakeParseArguments)
 
 # add a new target which is a SFML library
 # ex: sfml_add_library(sfml-graphics
-#                      SOURCES sprite.cpp image.cpp ...
-#                      DEPENDS sfml-window sfml-system
-#                      EXTERNAL_LIBS opengl freetype ...)
```

Comment:
Distinction was only for internal vs external dependencies? (linking against a lib already creates a dependency)
I removed these because it doesn't allow specifying dependency by dependency it that said dependency is PRIVATE or PUBLIC